spec/libraries/azure_engine_spec.rb

```diff
@@ -13,7 +13,10 @@ def create_message(queue_name, message)
       end
 
       def list_messages(queue_name, timeout, options = {})
-        []
+        # []
+        # msg.id, msg.message_text, msg.pop_receipt
+        q = Entity::Queue::DummyMessage.new
```

Comment:
__[reek]__
* [UncommunicativeVariableName] has the variable name 'q' [:ref](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md)

<!-- Comment by [SideCI](https://sideci.com) -->